extra_model/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 from extra_model._models import ExtraModel, extra_factory  # noqa
 
-__version__ = "0.1.0"
+__version__ = "0.1.0rc0"
```

Comment:
Current version we have in internal repo is 0.1.7. Should we start with 0.2.0 here just to avoid any possible issues with versions?

Reply:
yeah excellent point, thanks for the thorough review!